src/commands/createFunctionApp/functionStacks.ts

```diff
@@ -577,7 +617,7 @@ const windowsFunctionsStacks: string = `{
                             "javaVersion": "11",
                             "use32BitWorkerProcess": true
                         },
-                        "isPreview": true,
+                        "isPreview": false,
```

Comment:
Do you need to change isDefault for this as well? (L570)

Reply:
idk - `isDefault` has no effect in VS Code anyways. We always prompt for a runtime version, even in "basic" create.

Reply:
And again - I didn't "change" these myself. I just copied it from the linked repo in the description.